lib/commands/view.js

```diff
@@ -166,6 +166,15 @@ class View extends BaseCommand {
       log.info('view', `Ignoring invalid version: ${v}`)
       return false
     }).sort(semver.compareLoose)
+    const pckmntTime = {}
+    const invalidVersions = Object.keys(pckmnt.time).filter(v => !semver.valid(v))
+    invalidVersions.forEach(v => {
+      pckmntTime[v] = pckmnt.time[v]
+    })
+    pckmnt.versions.forEach(v => {
+      pckmntTime[v] = pckmnt.time[v]
+    })
+    pckmnt.time = pckmntTime
```

Comment:
this also filters out invalid versions, and shouldn't.

probably better would be using `Object.entries(pckmnt.time)` and `.sort`ing it using `semver`?

Reply:
Thanks for reviewing. But I collected the invalid version at line 170.

Reply:
it still seems simpler and harder to mess up to me, to do
```suggestion
    pckmnt.time = Object.fromEntries(Object.entries(pckmnt.time).sort(semver.compare))
```

Reply:
This looks very neat and clear, thank you!